backend/git_real/activity_log_creators.py

```diff
@@ -46,7 +46,7 @@ def log_pr_opened(pull_request):
     match = LogEntry.objects.filter(
         actor_user=pull_request.user,
         effected_user=pull_request.user,
-        object_1=pull_request,
+        # object_1=pull_request,
```

Comment:
do you also get this error if you have object_1 = pull_request?:
```
django.core.exceptions.FieldError: Field 'object_1' does not generate an automatic reverse relation and therefore cannot be used for reverse querying. If it is a GenericForeignKey, consider adding a GenericRelation.
```

Reply:
In terms of object 2, I'm not fully sure about it - sometimes it is there and sometimes it isn't